.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   - v12.10.0
 script:
-  - npm test
+  - run-s test docs check-clean
```

Comment:
This is to prevent checked in docs that remark would alter as soon as it ran.

Reply:
And it seems to work.
https://travis-ci.org/snabbdom/snabbdom/builds/617412084#L306-L315